test/functional/p2p_tx_download.py

```diff
@@ -121,38 +116,10 @@ def test_inv_block(self):
         # * the first time it is re-requested from the outbound peer, plus
         # * 2 seconds to avoid races
         assert self.nodes[1].getpeerinfo()[0]['inbound'] == False
-        timeout = 2 + (MAX_GETDATA_RANDOM_DELAY + INBOUND_PEER_TX_DELAY) + (
-            GETDATA_TX_INTERVAL + MAX_GETDATA_RANDOM_DELAY)
+        timeout = 2 + INBOUND_PEER_TX_DELAY + GETDATA_TX_INTERVAL
         self.log.info("Tx should be received at node 1 after {} seconds".format(timeout))
         self.sync_mempools(timeout=timeout)
 
-    def test_in_flight_max(self):
```

Comment:
Should replace this test with one that checks we start applying `OVERLOAD_PEER_TX_DELAY` ?

Reply:
@ajtowns Feel like writing such a test?

Reply:
Done.